testing/integration/integration.go

```diff
@@ -0,0 +1,112 @@
+// Package integration performs initialization and validation for integration
+// tests.
+package integration
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+)
+
+// ExecuteGollum execute gollum binary for integration testing
+func ExecuteGollum(config string, inputs []string, arg ...string) (out bytes.Buffer, err error) {
+	if config != "" {
+		arg = append(arg, "-c="+getTestConfigPath(config))
+	}
+
+	var stdin io.WriteCloser
+	timeout := 2 * time.Second
+	hasInputValues := len(inputs) > 0
+
+	cmd := GetGollumCmd(timeout, arg...)
+	cmd.Stdout = &out
+
+	if hasInputValues {
+		stdin, err = cmd.StdinPipe()
+		if err != nil {
+			return
```

Comment:
I personally prefer complete returns over automatic ones because of variable shadowing / unwanted assignments.